Views/ValuesField.cshtml

```diff
@@ -0,0 +1,24 @@
+@model Etch.OrchardCore.Fields.Values.ViewModels.DisplayValuesFieldViewModel
+
+@using OrchardCore.Mvc.Utilities;
+@using Etch.OrchardCore.Fields.Values.Settings
+
+@{
+    var modifier = Model.PartFieldDefinition.Name.HtmlClassify();
+    var settings = Model.PartFieldDefinition.Settings.ToObject<ValuesFieldSettings>();
+}
+
+@if (Model.Data.Any())
+{
+    <ul class="list list--@modifier">
+    @foreach (var item in Model.Data)
+    {
+        <li class="list__item">@item</li>
+    }
+    </ul>
+}
+
+@if (!Model.Data.Any() && string.IsNullOrWhiteSpace(settings.EmptyMessage))
+{
+    <p>@settings.EmptyMessage</p>
```

Comment:
Shouldn't this be using `DefaultValueFieldSettings.EmptyMessage` if `settings.EmptyMessage` isn't available? Though I suppose then there isn't a way to opt out of having an `EmptyMessage`.

Reply:
That was my thinking.